web/cobrands/sass/_base.scss

```diff
@@ -496,7 +496,7 @@ ul.error {
   display: block;
   width: 175px;
   height: 60px;
-  background: url($image-sprite) -8px 5px no-repeat;
+  background: transparent url('images/site-logo.png') 50% 50% no-repeat;
```

Comment:
Is this the same as site-logo-default in fixmystreet.com cobrand? Should we move that to the base now?

Reply:
`cobrands/fixmystreet/images/site-logo.png` and `cobrands/fixmystreet.com/images/site-logo-default.png` are identical, yes.

Are you suggesting we change `.site-logo__fallback` in `cobrands/fixmystreet.com/base.scss` to use `svg-background-image("/cobrands/fixmystreet/images/site-logo");` (and then move `cobrands/fixmystreet.com/images/site-logo-default.svg` to `cobrands/fixmystreet/images/site-logo.svg` to match)? And then, since the base cobrand would have an svg version of the logo at its disposal, use that with `svg-background-image()` in `sass/_base.scss` too?

_Or_ are you suggesting we do all the fancy inline SVG stuff in the base cobrand?

Reply:
I was suggesting using SVG in the base, ignoring the front page stuff (which would cover most of the .com-only stuff, I think?).
We could also do the inline stuff in base and wrap the insert in a TRY (same a I did the critical CSS) so it works if it's there, not if it isn't.